common/src/main/java/com/wynntils/utils/mc/McUtils.java

```diff
@@ -104,7 +105,31 @@ public static void sendCommand(String command) {
         mc().getConnection().sendCommand(command);
     }
 
-    public static void sendChat(String command) {
-        mc().getConnection().sendChat(command);
+    /**
+     * Sends some chat message directly to the server.
+     * Does not respect ChatTabFeature settings.
+     * @param message The message to send.
+     */
+    public static void sendChat(String message) {
+        mc().getConnection().sendChat(message);
+    }
+
+    /**
+     * Simulates the player typing a message in chat. Respects ChatTabFeature settings.
+     * @param message The message to send.
+     */
+    public static void sendChatAsPlayer(String message) {
```

Comment:
McUtils methods should absolutely not have any knowledge about Services or any other component types.

Reply:
Agreed. It seems we have created a bit of a messy system for ourselves, if trade market, chat tabs and wynnic transcriptions all are intermingled. Ideally, we should try to tease them apart.